ironfish/src/sdk.test.ts

```diff
@@ -152,27 +151,6 @@ describe('IronfishSdk', () => {
       expect(ipc?.namespaces).toEqual(ALL_API_NAMESPACES)
     })
 
-    it('should use secure RPC namespaces for TCP', async () => {
-      const sdk = await IronfishSdk.init({
-        dataDir: os.tmpdir(),
-        configOverrides: {
-          enableRpcTcp: true,
-          enableRpcTls: false,
-        },
-      })
-
-      const node = await sdk.node()
-      const tcp = node.rpc.adapters.find<RpcTcpAdapter>(
-        (a): a is RpcTcpAdapter => a instanceof RpcTcpAdapter,
-      )
-
-      const allowedNamespaces = ALL_API_NAMESPACES.filter(
-        (namespace) => !API_NAMESPACES_PROTECTED.includes(namespace),
-      )
-
-      expect(tcp?.namespaces.sort()).toMatchObject(allowedNamespaces.sort())
-    })
-
     it('should use all RPC namespaces for TCP with rpcTcpSecure flag', async () => {
```

Comment:
This test should be looked it, it refers to a deleted flag.

Reply:
changed the name